tests/test_subset.py

```diff
@@ -1811,7 +1812,7 @@ def test_temporal_he5file_subset(data_dir, subset_output_dir):
             if 'BRO' in i:
                 assert any('utc' in x.lower() for x in time_var_names)
 
-            dataset, _ = subset.convert_to_datetime(dataset, time_var_names)
+            dataset, _ = subset.convert_to_datetime(dataset, time_var_names, i[1].split('.')[-1])
```

Comment:
For getting extensions, it seems to me like we should use standard library methods to avoid edge cases (e.g., see https://stackoverflow.com/a/541394).  Using the `os` library would probably be the simplest and most straightforward here. (although `pathlib` seems to me to be the most robust, I think it would require a larger refactoring)